.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,31 @@
 ---
-name: Bug report
-about: Describe a scenario in which this project behaves unexpectedly
+name: "Bug report \U0001F41B"
+about: Create a report to help us improve
 title: ''
-labels: bug, needs-triage
+labels: bug
 assignees: ''
 
 ---
 
 [NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )
 
-## Description
-
+**Description**
 [NOTE]: # ( Describe the problem you're encountering. )
-[TIP]:  # ( Do NOT give us access or passwords to your New Relic account or API keys! )
-
-## Steps to Reproduce
-
-[NOTE]: # ( Please be as specific as possible. )
-
-## Expected Behaviour
+[TIP]:  # ( Do NOT share sensitive information, whether personal, proprietary, or otherwise! )
 
+**Expected Behavior**
 [NOTE]: # ( Tell us what you expected to happen. )
 
-## Relevant Logs / Console output
+**[Troubleshooting](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) or [NR Diag](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) results**
+[NOTE]: # ( Provide any other relevant log data such as the agent and/or profiler logs, preferably at DEBUG level or higher. )
+[TIP]:  # ( Scrub logs and diagnostic information for sensitive information )
 
-[NOTE]: # ( Please provide specifics of the local error logs, Browser Dev Tools console, etc. if appropriate and possible. )
-
-## Your Environment
-
-[TIP]:  # ( Include as many relevant details about your environment as possible. )
-
-* ex: Browser name and version:
-* ex: Operating System and version:
+**Steps to Reproduce**
+[NOTE]: # ( Please be as specific as possible. )
+[TIP]:  # ( Link a sample application that demonstrates the issue. )
 
-## Additional context
+**Your Environment**
+[TIP]:  # ( Include as many relevant details about your environment as possible including the running version of the agent and any relevant configurations, such as which .NET framework or core version your apllication is running on. )
```

Comment:
"such as which .NET framework or core version your apllication is running on. )"
=>
"such as the .NET framework or core version of your application.)"

Reply:
Good idea, and thanks for catching my typo.  "Apllication?" 🤦